universal-login-sdk/lib/api/DeployedWallet.ts

```diff
@@ -39,29 +40,29 @@ export class DeployedWallet implements ApplicationWallet {
     };
   }
 
-  async addKey(publicKey: string, transactionDetails: Partial<Message>): Promise<Execution> {
-    return this.sdk.addKey(this.contractAddress, publicKey, this.privateKey, transactionDetails);
+  async addKey(publicKey: string, paymentOptions: Partial<PaymentOptions> = {}): Promise<Execution> {
```

Comment:
Why `= {}`?

Reply:
Because in this scenario we can use this function more generic. Someone using sdk can pass there all paymentOptions, part of paymentOptions or pass nothing and sdk will provide missing parameters.